src/components/config/parse-postman.ts

```diff
@@ -23,65 +23,75 @@
  **********************************************************************************/
 
 import { Config } from '../../interfaces/config'
-import { Probe } from '../../interfaces/probe'
 import { DEFAULT_THRESHOLD } from '../../looper'
 
-let probes: Probe[] = []
+type CollectionVersion = 'v2.0' | 'v2.1'
 
-const getConvertedProbeFromPostmanItem = (item: any) => {
-  const req = item.request
-  const probe: Probe = {
-    id: item.name,
-    name: item.name,
-    requests: [
-      {
-        id: req.name,
-        url: req.url.raw,
-        method: req.method,
-        headers: req?.header?.reduce(
-          (obj: any, it: any) => Object.assign(obj, { [it.key]: it.value }),
-          {}
-        ),
-        body: req?.body?.raw || JSON.parse('{}'),
-        timeout: 10_000,
-      },
-    ],
-    incidentThreshold: DEFAULT_THRESHOLD,
-    recoveryThreshold: DEFAULT_THRESHOLD,
-    alerts: [],
+const getCollectionVersion = (config: any) => {
+  if (config?.info?.schema?.includes('v2.0')) {
+    return 'v2.0'
   }
 
-  return probe
+  if (config?.info?.schema?.includes('v2.1')) {
+    return 'v2.1'
+  }
+
+  throw new Error('UnsupportedVersion')
```

Comment:
Seeing this, do we need to add an information to the docs saying that only v2.0 and v2.1 collection are supported?

Reply:
Sure, I'll add the information to the docs.